pandas/tests/io/excel/conftest.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+
+import pandas.util.testing as tm
+
+from pandas.io.parsers import read_csv
+
+
+@pytest.fixture
+def frame(float_frame):
+    return float_frame[:10]
+
+
+@pytest.fixture
+def tsframe():
```

Comment:
This is only used in test_writers so could move there as well. Not sure if we care to make that level of distinction between modules and conftest

Reply:
i'm OK with having any discussion of fixture location separate to this PR.